### PR TITLE
Update SIG Release meeting times

### PR DESCRIPTION
**Pull request by xmudrii**
As pointed out by @helayoty (thanks a lot!), SIG Release meeting times are incorrect now that daylight saving started. It's supposed to be 14:30 UTC and 17:00 UTC altering every week.

/assign @saschagrunert @cpanato @puerco 
cc @kubernetes/sig-release-leads

### CODE REVIEW

File: sig-release/README.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ The [charter](charter.md) defines the scope and governance of the Release Specia
...
-* Regular SIG Meeting APAC/EU Oriented Time: [Tuesdays at 15:30 UTC](https://zoom.us/j/327142148?pwd=RE9aZWtCc0hhOWs4ZTdXZ0hBM0ROdz09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=15:30&tz=UTC).
+* Regular SIG Meeting APAC/EU Oriented Time: [Tuesdays at 14:30 UTC](https://zoom.us/j/327142148?pwd=RE9aZWtCc0hhOWs4ZTdXZ0hBM0ROdz09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=15:30&tz=UTC).
   * [Meeting notes and Agenda](https://bit.ly/k8s-sig-release-meeting).
   * [Meeting recordings](https://bit.ly/k8s-sig-release-videos).
-* Regular SIG Meeting US Oriented Time: [Tuesdays at 17:00 UTC](https://zoom.us/j/327142148?pwd=RE9aZWtCc0hhOWs4ZTdXZ0hBM0ROdz09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:00&tz=UTC).
+* Regular SIG Meeting US Oriented Time: [Tuesdays at 16:00 UTC](https://zoom.us/j/327142148?pwd=RE9aZWtCc0hhOWs4ZTdXZ0hBM0ROdz09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:00&tz=UTC).
```
Comment (mrbobbytables) [inline sig-release/README.md:20]: You'll need to make these changes in sigs.yaml in the root of the repo and then run `make`. It'll then update times in the various docs 👍

Comment (xmudrii) [reply sig-release/README.md]: Thanks for letting me know! It should be fixed now.

**Review by saschagrunert — APPROVED** [review]
Thank you for the update!

/lgtm

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

thanks

**Pull request merged**